airflow-core/src/airflow/exceptions.py

```diff
@@ -26,6 +26,9 @@
 from http import HTTPStatus
 from typing import TYPE_CHECKING, Any, NamedTuple
 
+# Import from shared library for backward compatibility
+from airflow._shared.configuration.exceptions import AirflowConfigException as _AirflowConfigException
```

Comment:
```suggestion
from airflow._shared.configuration.exceptions import AirflowConfigException as AirflowConfigException
```

And then you don't need the extra sublcass. The `X as X` is an accepted way to say "we are re-exporting" this class, and ruff/typecheckers etc won't complain about unnecessary import either.

Reply:
Yeah, I thought about it a bit more and that sounds OK to me, made a change and pushed via: fcd47c9e86